bricks/flutter_clean_base/__brick__/{{project_name.snakeCase()}}/lib/app.gen.dart

```diff
@@ -19,18 +21,21 @@ import 'package:flutter_gen/gen_l10n/app_localizations.dart';{{/handLocalization
 {{^handLocalization}}import 'package:{{project_name}}/app/localization/generated/l10n.dart';{{/handLocalization}}
 {{^isGoRouter}}{{#sentry}}import 'package:sentry_flutter/sentry_flutter.dart';{{/sentry}}{{/isGoRouter}}
 
-class App extends StatefulWidget {
+class App extends {{#isRiverpod}}Consumer{{/isRiverpod}}StatefulWidget {
```

Comment:
@cozvtieg9, maybe we should take out app.dart files for different state managers? With each new gate manager they become unreadable. They all have different controllers for ThemeMode, they create them differently. Only the structure is common and the routes